packages/common/src/initiative-templates/HubInitiativeTemplate.ts

```diff
@@ -156,7 +153,7 @@ export class HubInitiativeTemplate
    */
   async getEditorConfig(
     i18nScope: string,
-    type: EditorType
+    type: EntityEditorType
```

Comment:
I think you could make this more explicit with `InitiativeTemplateEditorType` since this is a method on the `HubInitiativeTemplate` class